### PR TITLE
STORY-8818: add bitfield helpers

### PR DESCRIPTION
**Pull request by gabe-kent**

### CODE REVIEW

File: lib/aggregate/attribute/bitfield.rb [hunk 1]
```diff
@@ -42,6 +42,10 @@ def validation_errors(_value)
 
   private
 
+  def value_to_string(value)
+    value.is_a?(Array) ? value&.map { |v| klass.to_bit(v) }&.join : value&.to_s
```
Comment (ColinDKelley) [inline lib/aggregate/attribute/bitfield.rb:46]: To be a bit more general with other enumerable types, maybe just check that it respond to `map`? And no need for `&` when it does:
```
value.respond_to?(:map) ? value.map { |v| klass.to_bit(v) }.join : value&.to_s
```

Comment (gabe-kent) [reply lib/aggregate/attribute/bitfield.rb:46]: Thanks, Colin! 👍

Comment (ColinDKelley) [reply lib/aggregate/attribute/bitfield.rb]: Looks good! 👍

**Review by ColinDKelley — APPROVED** [review]

**Review by primiti — APPROVED** [review]
Nice

**Review by victorborda — APPROVED** [review]

**Pull request merged**